.changeset/big-dolls-behave.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog': minor
+'@backstage/core-components': minor
```

Comment:
Please split the `Header` change into a separate changeset, it needs its own working. I'll be a `patch` one too, since that package is on `0.x`.